### PR TITLE
Chalice Native App Quickstart

### PR DESCRIPTION
**Pull request by Typettrey**

### CODE REVIEW

**Comment by sfc-gh-lambrosetti** [comment]
Hi @Typettrey  - thanks for submitting the guide! Overall, this will work, but we'll need a few changes:

The major thing is that the "Metadata Configuration" page (your page 2) shouldn't be showing at all. This is the relevant snippet I'm talking about:

```
<!-- ------------------------ -->
## Metadata Configuration
Duration: 2

It is important to set the correct metadata for your Snowflake Guide. The metadata contains all the information required for listing and publishing your guide and includes the following:


- **summary**: A concise guide to demonstrate use of the Chalice TVB Native App to detect overbidding in programmatic auctions. 
- **id**: chalice-native-app-quickstart
- **categories**: data-science, native-app
- **environments**: web 
- **status**: Published
- **feedback link**: https://github.com/Snowflake-Labs/sfguides/issues
- **tags**: Getting Started, Data Science, Chalice, Native App, Machine Learning 
- **authors**: Tylynn Pettrey, Tucker Ward
```

instead, it should be at the top of the markdown file like this:

```
summary: A concise guide to demonstrate use of the Chalice TVB Native App to detect overbidding in programmatic auctions. 
id: chalice-native-app-quickstart
categories: data-science, native-app
environments: web
status: Published
feedback link: https://github.com/Snowflake-Labs/sfguides/issues
tags: Getting Started, Data Science, Chalice, Native App, Machine Learning
authors: Tylynn Pettrey, Tucker Ward

# Chalice Native App Guide
<!-- ------------------------ -->

## Overview 
Duration: 1

Chalice True Value Bidding is a tool that learns from your historical spend
what the right price to pay for your inventory is. 

### Prerequisites
- Familiarity with Snowflake Data Loading Best Practices
- Access to the Trade Desk UI for Simulation Campaign

### What You’ll Learn 
- How to download a winrate report from The Trade Desk 
- How to load the resulting reports in a snowflake table  
- How to permission the table for the Chalice TVB Simulation App
- How to run the Chalice TVB Simulation App and reveal potential savings for your programmatic campaigns

### What You’ll Need 
- A Snowflake Account 
- Active Trade Desk Campaign UI Access
-Familiarity with Python 

### What You’ll Build 
- Load relevant reporting data to generate a simulation report to detect overbidding in programmatic auctions

<!-- ------------------------ -->
## Generate win-rate reports from The Trade Desk My Reports
Duration: 10

This will teach you how to create and download win-rate reports using the Trade Desk My Reports tools. Win-rate reporting allows the TVB Native App to approximate how much you are overpaying for programmatic advertising impressions. 
```

Other than that:
- the last heading should be "Conclusion and Resources"
- in the "Resources" section at the very end, please include a link to the app in the marketplace, a link to the Chalice profile in the marketplace, and/or other resources like Chalice's main site.

**Review by Typettrey — COMMENTED** [review]
Updated with the requested changes. Please let me know if anything else is needed!

**Comment by sfc-gh-lambrosetti** [comment]
hey @Typettrey - the metadata still isn't reading quite right - if you use the following instead, I was able to test and it works!

```
summary: "A concise guide to demonstrate the use of the Chalice TVB Native App to detect overbidding in programmatic auctions."
id: "chalice-native-app-quickstart"
categories: data-science, native-app
environments: "web"
status: "Published"
feedback_link: "https://github.com/Snowflake-Labs/sfguides/issues"
tags: Getting Started, Data Science, Chalice, Native App 
authors: Tylynn Pettrey, Tucker Ward 
```

**Comment by sfc-gh-lambrosetti** [comment]
hi @Typettrey - sorry, but I'm still not seeing these changes

we need this block at the top of the markdown file instead of what's there today (also, please no dashed lines before and after!)

```
summary: "A concise guide to demonstrate the use of the Chalice TVB Native App to detect overbidding in programmatic auctions."
id: "chalice-native-app-quickstart"
categories: data-science, native-app
environments: "web"
status: "Published"
feedback_link: "https://github.com/Snowflake-Labs/sfguides/issues"
tags: Getting Started, Data Science, Chalice, Native App 
authors: Tylynn Pettrey, Tucker Ward 
```

If it's easier, feel free to give me write access and I'm happy to help make the necessary changes!

**Comment by sfc-gh-lambrosetti** [comment]
@iamontheinet - this now runs locally and looks good - thank you!

**Comment by Typettrey** [comment]
Great, thanks for confirming!
________________________________

File: site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+summary: "A concise guide to demonstrate the use of the Chalice TVB Native App to detect overbidding in programmatic auctions."
+id: "chalice-native-app-quickstart"
+categories: data-science, native-app
+environments: "web"
+status: "Published"
+feedback_link: "https://github.com/Snowflake-Labs/sfguides/issues"
+tags: Getting Started, Data Science, Chalice, Native App 
+authors: Tylynn Pettrey, Tucker Ward 
+
+# Chalice Native App Guide
```
Comment (iamontheinet) [inline site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md:10]: @Typettrey Change the title to "Getting Started With Chalice Native App in Snowflake"

File: site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+- Familiarity with Snowflake Data Loading Best Practices
+- Access to the Trade Desk UI for Simulation Campaign
+### What You’ll Learn 
+- How to download a winrate report from The Trade Desk 
+- How to load the resulting reports in a snowflake table  
+- How to permission the table for the Chalice TVB Simulation App
+- How to run the Chalice TVB Simulation App and reveal potential savings for your programmatic campaigns
+
+### What You’ll Need 
+- A Snowflake Account
```
Comment (iamontheinet) [inline site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md]: @Typettrey Add link to Snowflake signup/signin page.

File: site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+- Access to the Trade Desk UI for Simulation Campaign
+### What You’ll Learn 
+- How to download a winrate report from The Trade Desk 
+- How to load the resulting reports in a snowflake table  
+- How to permission the table for the Chalice TVB Simulation App
+- How to run the Chalice TVB Simulation App and reveal potential savings for your programmatic campaigns
+
+### What You’ll Need 
+- A Snowflake Account 
+- Active Trade Desk Campaign UI Access
```
Comment (iamontheinet) [inline site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md]: @Typettrey Is there a link to the UI that be added here?

**Review by iamontheinet — CHANGES_REQUESTED** [review]

**Comment by iamontheinet** [comment]
@Typettrey Pl resolve the conflicts.

**Pull request closed without merging**